website/docs/docs/integrate/quickstart-semantic-layer.md

```diff
@@ -0,0 +1,292 @@
+sidebar_label: "Quickstart"
+---
+
+# dbt Semantic Layer quickstart
+
+:::info 📌
+
+The dbt Semantic Layer is currently available for public preview! With the dbt Semantic Layer, you'll be able to centrally define business metrics, remove code duplication and inconsistency, create self-service in downstream tools, and more! 
+
+
+For more info on public preview and plan availability, check out the [public preview](/docs/integrate/quickstart-semantic-layer#public-preview) and [product architecture](/docs/integrate/dbt-semantic-layer#product-architecture) section.
+
+:::
+
+
+## Public preview
+    
+We're excited to announce the dbt Semantic Layer is currently available for public preview, which means:
+
+&mdash; **Who?** The dbt Semantic Layer is open to all dbt Cloud tiers (Developer, Team, and Enterprise) during public preview. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse dbt metrics in external tools, which requires access to the Metadata API. For more info on plan availability, check out our [product architecture](/docs/integrate/dbt-semantic-layer#product-architecture) section.
+
+&mdash; **What?** Public preview provides early access to new features, is supported and production ready, but not priced yet. Pricing for the dbt Semantic Layer will be introduced alongside the General Available (GA) release. There may also still be additions or modifications to product behavior. 
+
+&mdash; **When?** Public preview will end once the dbt Semantic Layer is available for GA. After GA, the dbt Semantic Layer will only be available to dbt Cloud **Team** and **Enterprise** plans.
+
+&mdash; **Where?** Public preview is enabled at the account level so you don’t need to worry about enabling it per user.
+
+&mdash; **Why?** Public preview is designed to test the functionality and collect feedback from the community on performance, usability, and documentation. 
+
+
+## Introduction
+
+To try out the features of the dbt Semantic Layer, you first need to have a dbt project set up. This quickstart guide will lay out the following steps, and recommends a workflow that demonstrates some of its essential features:
+
+- Install dbt metrics package
+- Define metrics
+- Query, and run metrics
+- Configure the dbt Semantic Layer
+
+## Prerequisites
+To use the dbt Semantic Layer, you’ll need to meet the following:
+
+- Have a multi-tenant [dbt Cloud](https://cloud.getdbt.com/) Teams or Enterprise account. 
+   * Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API.
+- Have both your production and development environments running dbt version 1.2 (latest) or higher
+- Use Snowflake data platfrom 
+- Install the [dbt metrics package](https://hub.getdbt.com/dbt-labs/metrics/latest/) version 0.3.2 or higher in your dbt project
```

Comment:
Can we version this section and the dbt version? I think it could be confusing and lead to version mismatch if they try and use 0.3.2 with dbt core 1.3